### ISSUE TITLE
Scrollable content [0ssw9k]: Equal Access checker listed as inconsistent because it doesn't report additional AAA criteria

### ISSUE THREAD

**Issue by tombrunet** [auto-wcag/auto-wcag#2198]
For https://www.w3.org/WAI/standards-guidelines/act/rules/0ssw9k/

The Equal Access Accessibility Checker is listed as "Partial", but all of the results, and the AA mappings that the EA Checker supports are all correct. Therefore, the reports given are, in fact, consistent:
https://www.w3.org/WAI/standards-guidelines/act/implementations/equal-access/#breakdown-0ssw9k

My initial reaction was that the AAA SC should be listed as secondary, however, it's not like the contrast scenario that is more strict for the scenario being tested, but rather the rule doesn't address the aspects of 2.1.3 that happen to be more strict than 2.1.1. Therefore, there aren't pass scenarios that will fail 2.1.3. Should the report ignore AAA mappings for tools that don't advertise support for AAA (when everything else is consistent), or should this checkpoint be marked as secondary somehow, and if so, with what justification?

**Comment by philljenkins** [comment]
ACT is also incorrect reporting of AAA SC's in ACT rule `bisz58` [Meta element has no refresh delay (no exception)](https://www.w3.org/WAI/standards-guidelines/act/implementations/equal-access/#breakdown-bisz58) by the Equal Access Accessibility Checker.

Because the Checker doesn't support AAA SC's it reports against the Secondary Requirements


> [2.2.1 Timing Adjustable (Level A)](https://www.w3.org/TR/WCAG21/#timing-adjustable): This success criterion is less strict than this rule. This is because this criterion allows redirects longer than 20 hours. Some of the failed examples satisfy this success criterion.
